test/tests/functional/pbs_reservations.py

```diff
@@ -1327,53 +1328,47 @@ def test_standing_resv_with_job_array(self):
         exp_attr = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2")}
         self.server.expect(RESV, exp_attr, id=rid)
         rid_q = rid.split(".")[0]
-        # Submit a job-array within resrvation
-        j = Job(TEST_USER, attrs={ATTR_q: rid_q, ATTR_J: '1-5'})
-        j.set_sleep_time(40)
+        # Submit a job-array within reservation
+        j = Job(TEST_USER, attrs={'Resource_List.select': '1:ncpus=1',
+                                  ATTR_q: rid_q, ATTR_J: '1-4'})
+        j.set_sleep_time(15)
         jid = self.server.submit(j)
         subjid = []
-        for i in range(1, 5):
+        for i in range(1, 4):
             subjid.append(j.create_subjob_id(jid, i))
-        self.server.expect(JOB, {'job_state': 'Q',
-                                 'comment': (MATCH_RE,
-                                             'Queue not started')}, jid)
         # Wait for standing reservation first instance to start
         exp_attr = {'reserve_state': (MATCH_RE, "RESV_RUNNING|5")}
-        self.server.expect(RESV, exp_attr, id=rid, offset=10)
+        self.server.expect(RESV, exp_attr, id=rid)
         self.server.expect(RESV, {'reserve_index': 1}, id=rid)
         self.server.expect(JOB, {'job_state': 'B'}, jid)
         self.server.expect(JOB, {'job_state=R': 4}, count=True,
                            id=jid, extend='t')
-        self.server.expect(JOB, {'job_state=Q': 1}, count=True,
-                           id=jid, extend='t')
         # Wait for standing reservation first instance to finished
         self.logger.info(
-            'Waiting 60 seconds for first instance of reservation to Finish')
-        time.sleep(60)
+            'Waiting 20 seconds for first instance of reservation to Finish')
         exp_attr = {'reserve_state': (MATCH_RE, "RESV_CONFIRMED|2")}
-        self.server.expect(RESV, exp_attr, id=rid)
-
+        self.server.expect(RESV, exp_attr, id=rid, offset=20)
+        self.server.expect(JOB, 'queue', op=UNSET, id=jid)
         # Wait for standing reservation second instance to start
         self.logger.info(
-            'Waiting 60 seconds for second  instance of reservation to start')
-        self.server.expect(RESV, {'reserve_index': 2}, offset=60, id=rid)
+            'Waiting 50 sec for second  instance of reservation to start')
         exp_attr = {'reserve_state': (MATCH_RE, "RESV_RUNNING|5")}
-        self.server.expect(RESV, exp_attr, id=rid)
+        self.server.expect(RESV, exp_attr, id=rid, offset=50, interval=10)
+        self.server.expect(RESV, {'reserve_index': 2}, id=rid)
         # Wait for reservations to be finished
         msg = "Que;" + rid_q + ";deleted at request of pbs_server@"
         self.server.log_match(msg, starttime=end, interval=20)
```

Comment:
I'd suggest you reduce the interval to a second because the reservation is already running and it should get deleted  10 seconds from now.